sdk/compute/Azure.ResourceManager.Compute/src/Customize/VirtualMachineExtensionData.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Azure.Core;
+using Azure.ResourceManager.Compute.Models;
+
+namespace Azure.ResourceManager.Compute
+{
+    public partial class VirtualMachineExtensionData
+    {
+        /// <summary>
+        /// The extensions protected settings that are passed by reference, and consumed from key vault
+        /// <para>
+        /// To assign an object to this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
+        /// </para>
+        /// <para>
+        /// To assign an already formated json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public BinaryData ProtectedSettingsFromKeyVault
+        {
+            get
+            {
+                string raw;
```

Comment:
This doesn't look like the pattern we talked about last night?  Did we find out that it didn't work?

Reply:
Sorry I forgot we have multiple occurrences of this case. I will fix them